lib/src/package_analyzer.dart

```diff
@@ -192,7 +193,18 @@ class PackageAnalyzer {
 
     final licenses = await detectLicenseInDir(pkgDir);
     if (licenses.isNotEmpty) {
-      tags.add('license:${licenses.first.spdxIdentifier.toLowerCase()}');
+      tags.addAll(
+        licenses
+            .map((l) => l.spdxIdentifier)
+            .expand(
+              (id) => [
+                'license:${id.toLowerCase()}',
+                if (fsfLibreLicenses.contains(id)) 'license:fsf-libre',
+                if (osiApprovedLicenses.contains(id)) 'license:osi-approved',
+              ],
+            )
+            .toSet(),
```

Comment:
To `expand + toSet` seems a bit extreme... maybe it's simpler to do:
```suggestion
      tags.addAll(licenses.map((l) => 'license:${l.spdxIdentifier.toLowerCase()}').toSet());
      if (licenses.all((l) => fsfLibreLicenses.contains(l.spdxIdentifier))) {
        tags.add('license:fsf-libre');
      }
      if (licenses.all((l) => osiApprovedLicenses.contains(l.spdxIdentifier))) {
        tags.add('license:osi-approved');
      }
```

Also shouldn't we require all licenses to be OSI approved before we grant:`'license:osi-approved'`